speech/cloud-client/src/main/java/com/example/speech/Recognize.java

```diff
@@ -362,15 +380,15 @@ public static void streamingRecognizeFile(String fileName) throws Exception, IOE
     try (SpeechClient speech = SpeechClient.create()) {
 
       // Configure request with local raw PCM audio
-      RecognitionConfig recConfig = RecognitionConfig.newBuilder()
-          .setEncoding(AudioEncoding.LINEAR16)
-          .setLanguageCode("en-US")
-          .setSampleRateHertz(16000)
-          .setModel("default")
-          .build();
-      StreamingRecognitionConfig config = StreamingRecognitionConfig.newBuilder()
-          .setConfig(recConfig)
-          .build();
+      RecognitionConfig recConfig =
```

Comment:
It'd be helpful if you could do these formatting changes in a separate PR, so that there isn't this giant diff of unrelated changes for reviewers to review..

Reply:
Sorry, given my access issues, I'm not sure how it may affect creating a separate PR. Putting everything together in one for now.